frontend/staff-dashboard/src/pages/flexible_pricing/list.tsx

```diff
@@ -231,6 +262,16 @@ export const FlexiblePricingList: React.FC = () => {
                                                         <strong>Country of Income:</strong>
                                                         <div>{modaldata.country_of_income}</div>
                                                     </p>
+                                                    <p>
+                                                        <span>
+                                                            <strong>Justification:</strong>
+                                                        </span>
+                                                        <select onChange={(e) => handleChange(e)} style={{ marginLeft: "20px" }}>
+                                                            {All_Justifications.map((option) => (
+                                                            <option value={option.value} selected={justification == option.value}>{option.label}</option>
```

Comment:
Should display `option.value` here.

Reply:
<img width="725" alt="Screenshot 2022-06-15 at 4 04 43 PM" src="https://user-images.githubusercontent.com/7334669/173812576-a1481d4a-fcf0-4c05-a197-89d27fe195e1.png">